components/twitter/twitter.app.js

```diff
@@ -818,19 +885,212 @@ module.exports = {
         console.log(err) // TODO
```

Comment:
This is not a great pattern... but since you return undefined in these cases, you should just also return undefined in the 502 cases above

Reply:
don't try catch